docs/architecture/architecture.md

```diff
@@ -0,0 +1,169 @@
+# Algorithm Management Toolkit Architecture
+
+This document contains architectural decisions related to the Algorithm Managment Toolkit.
+
+## System Context
+
+The Algorithm Management Toolkit is an application that users can use to complete requirements
+that are specified by the algoritmekader. The diagram below sketches the broader system context
+of this Algorithm Management Toolkit.
+
+### Example
+Suppose a data science team is working on an ML-algorithm. A team member visits the Algoritmekader
+website and sees that among other things an IAMA is a required to be performed. The user selects
+the IAMA task from the Algoritmekader and is forwarded to the Algorithm Management Toolkit. Here
+the user can login and import the IAMA task. The Algorithm Management Toolkit imports the instructions
+on how to execute an IAMA and how to store the results from the Instrument Register. Now the user can
+perform the IAMA from within the Algorithm Management Toolkit. Relevant stakeholders can also login
+to the project page of the Algorithm Management Toolkit to answer questions from the IAMA. Relevant
+discussions can be captured within the toolkit as well. Upon completion the IAMA results are written
+to an Assessment Card within a System Card to a user specified location, usually a remote repository
+where the source code of the algorithm resides.
+
+
+```mermaid
+C4Context
+    title System Context diagram for Algorithm Management Toolkit
+    Boundary(b0, "Government of the Netherlands") {
+        System(Algoritmekader, "Algoritmekader", "Defines measures and instruments")
+        System(TAD, "Algorithm Management Toolkit", "Provides the execution of measures and instruments")
+        System(InstrumentRegister, "Instrument Register", "Contains information about how to execute instruments and measures")
+        System(Repository, "Repository", "Contains a System Card with filled in measures and instruments")
+    }
+
+    Person(user0, "User", "User wants to comply with regulations and required
+    ethical frameworks")
+
+
+    Rel(Algoritmekader, TAD, "Passes required measures and instruments")
+    UpdateRelStyle(Algoritmekader, TAD, $offsetY="50", $offsetX="-150")
+    UpdateLayoutConfig($c4ShapeInRow="3", $c4BoundaryInRow="1")
```

Comment:
i would maybe have shapeinrow=2 instead of 3, as the text is falling behind the boundary and also that we have everything except the AMT is not blue.

Reply:
good point :)